src/assets/images/flags/en.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
```

Comment:
change to english language flag with both US and GB flags, as we are not supporting BE